daemon/graphdriver/vfs/quota_unsupported.go

```diff
@@ -0,0 +1,18 @@
+// +build !linux
+
+package vfs
+
+type driverQuota struct {
+}
+
+func setupDriverQuota(driver *Driver) error {
+	return nil
+}
+
+func (d *Driver) setupQuota(dir string, size uint64) error {
+	panic("Platform does not support quota; this should never be called")
```

Comment:
Panic seems a little harsh here. Can we just return a not supported error?

Reply:
This should never be called. If this is called, it means something has gone wrong, or the code was changed in an unexpected way.